threadbuf.h

```diff
@@ -1,4 +1,5 @@
 // From http://stackoverflow.com/a/12413298
```

Comment:
Maybe remove the old code, if it is not used anymore ?

Reply:
Maybe in a second time, I just want to be sure that the new code is working well in navitia :)